former members page (#263)
sunpy/sunpy.org: project/former.rst

=======================

> Former Members of SunPy

Reviewer: People on this list might be still be active in the project, so this seems a little too final. "Emeritus role holders" or "previous role holders" or something?

Author: Changed to Emeritus role holders

=====================
Emeritus role holders
=====================

> This page lists the people who have served as official members of the SunPy Project in some capability.

Reviewer: ```suggestion
This page lists the people who have previously served in named roles in the SunPy Project.
```

.. list-table::
   :widths: 15, 60, 15
   :header-rows: 1

   * - Name
     - Role
     - Period
   * - `Andy Inglis <https://github.com/aringlis>`__
     - Board member and Secretary
     - 2014 - 2017
   * - `Albert Shih <https://github.com/ayshih>`__

>      - Board member
>      - 2014 - 2018

Author: ```suggestion
     - Board member, Lead Developer
     - 2014 - 2018, 2014 - 2015
```
